### ISSUE TITLE
Improve serials

### ISSUE THREAD

**Issue by cathoderaydude** [WinWorldPC/adventure#79]
Serials suck atm.
1) Add a DownloadUUID column (nillable) to tie a serial to a given download (windows 98 oem vs retail -_-)
2) Display any linked serials on the download info page
3) Add an ID PK to the Serials table and backfill :(
4) For anyone who's an admin, expose an AJAX insertion prompt in the serials field so we can quickly comb the comment threads

**Comment by NattyNarwhal** [comment]
for 1, that does mean we'd have to link it for foreign language copies too. 2. can be done through just 2. too, 3. is a simple script once the column is added (plus adding the trigger), 4. i assume you mean just have a field on the non-editing products page for it?

**Comment by cathoderaydude** [comment]
Yeah I guess we could just have a form that submits to the add serial route, no ajax bs needed?
